server.go

```diff
@@ -26,4 +31,7 @@ func main() {
 
 	log.Printf("connect to http://localhost:%s/ for GraphQL playground", port)
 	log.Fatal(http.ListenAndServe(":"+port, nil))
+
+	// Connect to database
+	pd := pokedex.NewPokedex()
```

Comment:
This is obviously wrong, but I'm not sure how to activate the database connection.